src/JBCCIPSucker.sol

```diff
@@ -54,6 +57,36 @@ contract JBCCIPSucker is JBSucker, ModifiedReceiver {
         REMOTE_CHAIN_ID = IJBCCIPSuckerDeployer(msg.sender).REMOTE_CHAIN_ID();
         REMOTE_CHAIN_SELECTOR = IJBCCIPSuckerDeployer(msg.sender).REMOTE_CHAIN_SELECTOR();
         ROUTER = IRouterClient(i_ccipRouter);
+        // TODO: Remove this init and revert this back to 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2 for prod
+        WETH = CCIPHelper.wethOfChain(block.chainid);
```

Comment:
should WETH address be passed into the constructor?

Reply:
not a huge diff for now.